.github/workflows/sync-plugin-registry.yml

```
@@ -38,7 +38,17 @@ jobs:
-    if: github.ref == 'refs/heads/logan/obsidian'
+    # Disabled 2026-07-10 per Logan's direct instruction: "anything currently
+    # writing swarm.json is eligible to be disabled." This job's `git add
+    # manifest.json swarm.json` push races the sync-agents-bootstrap.yml
+    # self-heal job on the same swarm.json-triggered push -- both jobs check
+    # out the same branch tip and push back, so the second push can be
+    # rejected non-fast-forward, leaving one artifact stale (flagged
+    # independently by Sourcery and GitHub Copilot on PR #831/#834, which
+    # were proposing this same job -- unaware it had already landed here).
+    # Left in place rather than deleted so the code is available if someone
+    # wants to fix the race (e.g. a shared concurrency group) and re-enable.
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)